### ISSUE TITLE
Add buildroot instructions?

### ISSUE THREAD

**Issue by traverseda** [ScottW514/meta-openglow#1]
I'm pretty new to embedded linux build environments, but I've found it a lot easier to get up and running with buildroot. Consider adding buildroot instructions, or a buildroot version?

**Comment by ScottW514** [comment]
I agree that Yocto has a _very_ steep learning curve.  However, all of the upstream projects that this project leverages are also based on Yocto, and my development path has been eased by sticking with it.  

I'll be documenting the layer stackup in the coming weeks.  I think this will help those without much Yocto experience to be able to ramp up quickly.  

If you find that Yocto still doesn't meet your needs, I'll be happy to assist if you want to put together a buildroot version. I have worked with buildroot, and found it fully capable.
